.circleci/config.yml

```diff
@@ -586,7 +605,12 @@ jobs:
       - run:
           name: 'Run minimal tests'
           command: |
-            gosu circleci make -C src/test/regress << parameters.make >> EXTRA_TESTS="$(for i in $(seq << parameters.runs >> | circleci tests split); do echo -n '<< parameters.test >> ' ; done)"
+            tests_array=($tests)
+            for test in "${tests_array[@]}"
+            do
+                echo "Running : $test on << parameters.make >>"
+                gosu circleci make -C src/test/regress << parameters.make >> EXTRA_TESTS="$(for i in $(seq << parameters.runs >> | circleci tests split); do echo -n $test '' ; done)"
```

Comment:
Afaict this is not smart yet about what make command it should use. So if you add a failure test this will run it with check-minimal instead of `check-failure-base`. I don't think we can merge this until it's smarter about that. Otherwise it will give a lot of noise.